.nvmrc

```diff
@@ -0,0 +1 @@
+6
```

Comment:
We only use LTS versions of node (4, 6, 8, 10 in the future), so Node 6 will still be supported until December 2019.